jscomp/others/js_json.ml

```diff
@@ -61,50 +61,66 @@ let test (type a) (x : 'a) (v : a kind) : bool =
   | Array -> Js_array.isArray x 
   | Object -> (Obj.magic x) != Js.null && Js.typeof x = "object" && not (Js_array.isArray x )
 
-let decodeString json = 
-  if Js.typeof json = "string" 
-  then Some (Obj.magic (json:t) : string)
+external unsafeParse : string -> t = "JSON.parse" [@@bs.val]
+
+let parse s =
+  let open Result in
+  try Ok (unsafeParse s) with
+  | e -> Error (Js.String.make e)
```

Comment:
where is `Result` introduced? if so, please make it `Bs_result` so that we can call it under `Bs.Result`